tr_sys/tr_ars/apps.py

```diff
@@ -66,5 +66,5 @@ def ready(self):
         # connect signals
         from . import signals
         logger.debug('### %s ready...' % (self.name))
-        post_migrate.connect(setup_schema, sender=self)
+        #post_migrate.connect(setup_schema, sender=self)
```

Comment:
I don't know enough why this would be commented out ... sorry for bein dumb here